### PR TITLE
NEPT-1783: Add missing inheritance in the theme calls.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: atomium/atomium.api.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ function hook_atomium_definition_hook() {
  */
 function hook_atomium_definition_form_hook($form, &$form_state) {
   $form['component'] = array(
-    '#theme' => 'username',
+    '#theme' => array('username'),
```
Comment (voidtek) [inline atomium/atomium.api.php:60]: why we do not use _atomium_extend_theme_hook function?

File: atomium/includes/common.inc [hunk 2]
```diff
@@ -496,13 +496,14 @@ function atomium_menu_get_active_breadcrumb() {
 
     foreach ($active_trail as $parent) {
       $breadcrumb[] = array(
-        '#theme' => array('link'),
+        '#theme' => array('link__breadcrumb'),
```
Comment (voidtek) [inline atomium/includes/common.inc]: why we do not use _atomium_extend_theme_hook function?

File: atomium/templates/indentation/indentation.component.inc [hunk 3]
```diff
@@ -24,18 +24,18 @@ function atomium_atomium_theme_indentation() {
...
-  if ($variables['size'] <= 0) {
-    return;
+  if (!is_numeric($variables['size']) || $variables['size'] <= 0) {
+    $size = 0;
   }
 
   $variables[$hook] = array(
     $hook => array(
-      '#theme' => array('item_list'),
+      '#theme' => _atomium_extend_theme_hook('item_list__size' . $size . '', $variables['theme_hook_original']),
```
Comment (voidtek) [inline atomium/templates/indentation/indentation.component.inc]: remove useless string.

File: atomium/templates/tablesort_indicator/tablesort_indicator.component.inc [hunk 4]
```diff
@@ -37,15 +36,19 @@ function atomium_preprocess_tablesort_indicator(&$variables, $hook) {
...
     $image['#alt'] = $image['#title'] = t('sort ascending');
+    $direction = 'asc';
   }
   else {
     $image['#path'] = 'misc/arrow-desc.png';
     $image['#alt'] = $image['#title'] = t('sort descending');
+    $direction = 'desc';
   }
 
+  $image['#theme'] = array('image__' . $direction . '', $variables['theme_hook_original']);
```
Comment (voidtek) [inline atomium/templates/tablesort_indicator/tablesort_indicator.component.inc]: Missing function call

File: atomium/atomium.api.php (hunk 1 above)
Comment (drupol) [reply atomium/atomium.api.php:60]: Because it's an example, do you think it worth it ?

File: atomium/includes/common.inc (hunk 2 above)
Comment (drupol) [reply atomium/includes/common.inc]: Thanks, fixed.

File: atomium/templates/indentation/indentation.component.inc (hunk 3 above)
Comment (drupol) [reply atomium/templates/indentation/indentation.component.inc]: Done.

File: atomium/templates/tablesort_indicator/tablesort_indicator.component.inc (hunk 4 above)
Comment (drupol) [reply atomium/templates/tablesort_indicator/tablesort_indicator.component.inc]: Fixed.

**Review by richardcanoe — APPROVED** [review]
Happy to approve this

**Pull request merged**